test/svg/parameters.py

```diff
@@ -758,6 +688,8 @@ def Gs(**kwargs):
     # discovered while investigatin issue 358:
     pytest.param(svgfile("issue_358b.svg"), id="repeated relative move"),
     pytest.param(svgfile("issue_358.svg"), id="arc start & initial point"),  # issue 358
+    pytest.param(svgfile("Ghostscript_colorcircle.svg"), id="ghostscript colorcircle"),
```

Comment:
Could you please also add tests in `test/svg/test_svg.py` that use those 2 files to produces PDFs, and check the generated file with `assert_pdf_equal`? _cf._ https://pyfpdf.github.io/fpdf2/Development.html#assert_pdf_equal-writing-new-tests

Reply:
Actually at first I added these tests:
```
class TestSVGStyleParsing:
    @pytest.mark.parametrize("svg_file", parameters.test_svg_sources[18:20])
    def test_draw_svg_style_to_page(self, tmp_path, svg_file):
        svg = fpdf.svg.SVGObject.from_file(svg_file)

        pdf = fpdf.FPDF(unit="pt", format=(svg.width, svg.height))
        pdf.add_page()
        svg.draw_to_page(pdf)

        assert_pdf_equal(pdf, GENERATED_PDF_DIR / f"{svg_file.stem}.pdf", tmp_path)
```
 but after running them I noticed that they were already executed with:
```
    @pytest.mark.parametrize("svg_file", parameters.test_svg_sources)
    def test_svg_conversion(self, tmp_path, svg_file):
        svg = fpdf.svg.SVGObject.from_file(svg_file)
```
So I removed them

Reply:
Oh you are totally right!
That's fine as it is then :)